### PR TITLE
[1/2] [R] sensors: Move Edo platform-specific configs to platform; update common configuration from PDX203 58.0.A.3.31

### PR DESCRIPTION
**Pull request by MarijnS95**
Depends on #851 

---
Files suffixed with `_somc_platform` - as the name implies - belong to the platform.  If more platforms with Kona were to exist they may have a different platform-specific overlay, which is not a problem now but important to maintain consistency going forward.

---
Imports files from Edo, and manually extends the list of SoCs with `"356"` and other new SoCs instead of blindly overwriting the list (which would sometimes remove other SoCs).

This, among which, fixes the ranges of the accelerometer adn gyroscope
being too small on Edo.

### CODE REVIEW

File: hardware/ssc/config/sns_device_orient.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
          "type": "flt", "ver": "0", "data": "0.25"
       },
       "param3": {
-         "type": "flt", "ver": "0", "data": "55.0"
+         "type": "flt", "ver": "0", "data": "70.0"
```
[the remark this answers is not in the document]
Comment (jerpelea) [reply hardware/ssc/config/sns_device_orient.json]: please remove this change

File: hardware/ssc/config/sns_device_orient.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
          "type": "flt", "ver": "0", "data": "2.0"
       },
       "param6": {
-         "type": "flt", "ver": "0", "data": "25.0"
+         "type": "flt", "ver": "0", "data": "15.0"
```
[the remark this answers is not in the document]
Comment (jerpelea) [reply hardware/ssc/config/sns_device_orient.json]: please remove this change

File: hardware/ssc/config/sns_device_orient.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarijnS95) [reply hardware/ssc/config/sns_device_orient.json]: These have already been reverted :) - didn't mark the comment as resolved but it says "outdated".

File: hardware/ssc/config/sns_device_orient.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (MarijnS95) [reply hardware/ssc/config/sns_device_orient.json]: Also already reverted because of the platform override.

**Pull request merged**